src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
-  margin: 0;
+@import './reset';
+@import './utils/mixins';
+
+.wrapper {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  box-sizing: initial;
+
+  &__minutes {
+    @include hand(20vmin, 3vmin, #0700ff);
+
+    transform: translate(-50%, -10vmin) rotate(-90deg);
+    animation-name: hand-move;
+    animation-duration: 3600s;
+    animation-iteration-count: infinite;
+    animation-timing-function: steps(60);
+  }
+
+  &__seconds {
+    @include hand(38vmin, 1.5vmin, #2c8000);
+
+    transform: translate(-50%, -19vmin) rotate(-90deg);
+    animation-name: hand-move;
+    animation-duration: 60s;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+  }
+
+  &__center {
+    @include hand(5vmin, 5vmin, #f6a603);
+
+    transform: translate(-50%, -1vmin);
+    border-radius: 50%;
+    z-index: 3;
+  }
+
+  &--speed-up {
+    & :nth-child(1) {
```

Comment:
better use `stopwatch__minute-hand` instead of` :nth-child(1) `because if you will change order of elements or add new element you will need also rewrite css

Reply:
))) 
This is like when you have tests and choose between two options)
First I have made nested classes)
But then I realized that we should not enhance classes specificity according to BEM)